src/administration/web.md

```diff
@@ -57,23 +65,33 @@ During a `merge`:
 
 Rebuilding the application is not necessary if the same code was already built (for any environment): in this case you will see the message ``Slug already built for this tree id, skipping``.
 
-------------------------------------------------------------------------
+After clicking `Merge` a dialog box will appear that will provide commands to execute future merges from the command line using the [Platform.sh CLI](/gettingstarted/cli.md).
+
+![Merge confirmation cli](/images/header_merge_box.png)
 
-![Sync](/images/icon-sync.png)
+#### Sync
 
-### Sync
+![Header Sync](/images/header_sync.png)
 
 Synchronizing means updating the child environment with the code and/or data of its parent and redeploying the child environment.
 
+After clicking `Sync` a dialog box will appear that will provide commands to execute future merges from the command line using the [Platform.sh CLI](/gettingstarted/cli.md).
```

Comment:
"Platform.sh CLI" only needs to link to the CLI page the first time it appears on a page, I'd argue.  Unless someone wants to debate me on that? :smile:

Reply:
No debates here on that. The only thing I can think of is if someone takes a link to a specific subsection (#), maybe it would be better to "link to Platform.sh CLI" only the first time in each subsection? Otherwise I get your point.